conf/machine/include/imx-base.inc

```diff
@@ -202,11 +202,11 @@ MACHINE_SOCARCH_FILTER_append_use-mainline-bsp = " \
     qtbase \
 "
 MACHINE_SOCARCH_FILTER_append_mx6q = " \
-    virtual/opencl-icd \
```

Comment:
Hmm, just noticed that `virtual/opencl-icd` is used by opencv recipe:

```
meta-openembedded/meta-oe/recipes-support/opencv/opencv_4.5.0.bb:PACKAGECONFIG[opencl] = "-DWITH_OPENCL=ON,-DWITH_OPENCL=OFF,opencl-headers virtual/opencl-icd,"
```

I see no provider of this though. Is this a mistake?

Reply:
Did you check other layers?

Reply:
I tried google but it doesn't find much. Only this secondary addition https://scuderia.cs.hs-rm.de/andreas.werner/am57xx-bsp/-/blob/1389e2cad79be1a98672168fa2d23904e5ed2d51/meta-hsrm/meta-hsrm-ti/recipes-ti/ocl/opencl_git.bbappend.

Reply:
But in this case we'd use the `opencl-icd-loader` right?

Reply:
I am aligning with `opencl-headers` and `opencl-icd-loader` and removing the virtual versions of these from our recipes. Then by setting the preferred version the `clinfo` recipe will use the `imx-gpu-viv` recipe instead of `opencl-headers` and `opencl-icd-loader` recipes.